test_external/eigen/Jamfile.v2

```diff
@@ -31,6 +31,6 @@ test-suite "odeint"
     [ run runge_kutta4.cpp ]
     [ run runge_kutta_dopri5.cpp ]
     [ run integrate.cpp ]
-    [ compile-fail fail_integrate.cpp ]
+    # [ compile-fail fail_integrate.cpp ]
```

Comment:
Any reason to not just delete this line?

Reply:
No. Done.